components/Form.js

```diff
@@ -152,6 +241,62 @@ export default function Form({
         onValueChange={handleCropTypeChange}
         labelButtonText={checkSelectInput(currentCropType, "crop type")}
       />
+      <Label htmlFor="image">Image</Label>
+      <ImageContainer>
+        <VisuallyHidden
+          type="file"
+          name="image"
+          id="fileInput"
+          accept="image/*"
+          onChange={handleImageUpload}
+          required
+        />
+
+        <CustomFileInputButton
+          htmlFor="fileInput"
+          tabIndex="0"
+          onClick={handleFileInputButtonClick}
+          onKeyDown={(event) => {
+            if (event.key === "Enter" || event.key === " ") {
+              event.preventDefault();
+              handleFileInputButtonClick(event);
+            }
+          }}
+        >
+          <SvgIcon variant="upload" size="80"></SvgIcon>
+          <SelectedFileName>Click to upload</SelectedFileName>
+        </CustomFileInputButton>
+
+        <ImagePreviewContainer>
+          {image ? (
+            <StyledImageWrapper>
+              <Image
+                src={URL.createObjectURL(image)}
+                alt="Preview of the image to upload"
+                sizes="300px"
+                fill
+                style={{
+                  objectFit: "contain",
+                }}
+              />
+            </StyledImageWrapper>
+          ) : (
+            <StyledImageWrapper>
+              <Image
+                src={data.image}
+                alt="Preview of the image to upload"
+                sizes="300px"
+                fill
+                style={{
+                  objectFit: "contain",
+                }}
+              />
+            </StyledImageWrapper>
+          )}
+          <SelectedFileName>{selectedName || "Preview"}</SelectedFileName>
+        </ImagePreviewContainer>
```

Comment:
We could potentially simplify this a little (these are quite a few things that are the same if image is truthy or falsy. In fact, the only difference i see is the src. If that is the case then we could do something like this.

``` js
<ImagePreviewContainer>
  <StyledImageWrapper>
    <Image
      src={image ? URL.createObjectURL(image) : data.image}
      alt="Preview of the image to upload"
      sizes="300px"
      fill
      style={{ objectFit: "contain" }}
    />
  </StyledImageWrapper>
  <SelectedFileName>{selectedName || "Preview"}</SelectedFileName>
</ImagePreviewContainer>
```